src/main/java/com/ryanharter/auto/value/moshi/AutoValueMoshiExtension.java

```diff
@@ -289,7 +289,11 @@ public MethodSpec createReadMethod(ClassName className,
 
       first = false;
     }
-    readMethod.endControlFlow(); // if/else if
+
+    readMethod.nextControlFlow("else");
+    readMethod.addStatement("$N.skipValue()", reader);
+
+    readMethod.endControlFlow(); // if/else if/else
```

Comment:
Ha, that's true... I've got confused reading it again...
Does `//if, else if, else` reads better?